### DISCUSSION TITLE
evaluate 的翻譯

### DISCUSSION THREAD

**Discussion by mattwang44** [python/python-docs-zh-tw#748] [Add Term]
### 初步確認

- [X] 我已經確認過 [術語列表](https://github.com/python/python-docs-zh-tw/wiki/%E8%A1%93%E8%AA%9E%E5%88%97%E8%A1%A8) 中沒有相關術語。
- [X] 我已經確認過 [Python 官方術語列表](https://docs.python.org/zh-tw/3/glossary.html) 中沒有相關術語。

### 術語原文

evaluate

### 術語翻譯

待討論

### 說明

我自己覺得跟「執行」、「計算」的意義都稍微不同，我大多會翻成「為...求值」（[glossary](https://docs.python.org/zh-tw/3/glossary.html#term-expression) 內也是）但也不是很通用，有些技術文章是用「評估」但我覺得用在軟體技術中並不是這個意思。

### 參考資料

軟體普遍常見的用詞，python doc 有多個出處，例如
```
#: ../../howto/perf_profiling.rst:100
msgid ""
"As you can see, the Python functions are not shown in the output, only "
"``_PyEval_EvalFrameDefault`` (the function that evaluates the Python "
"bytecode) shows up. Unfortunately that's not very useful because all Python "
"functions use the same C function to evaluate bytecode so we cannot know "
"which Python function corresponds to which bytecode-evaluating function."
```

**Comment by mindihx** [comment]
evaluate 可能在不同的情境下可以翻成不同的中文
在這個情境下，因為 python 的 eval(xxx) method 就是執行 xxx 的意思
所以覺得這裡的 evaluate 可以翻成執行

不過它的動作是執行，目的是求值，所以翻成「為...求值」也通

**Comment by ken71301** [comment]
3/30：原文中有的主要用法，例如 `If *exc_info* does not evaluate as false...` 此處的意義應更接近於` 給值` 而非 `求值`。
其原文 `to judge or calculate the quality, importance, amount, or value of something` 也有部分計算之意，因此目前需根據前後文暫定為 `給值` 或 `計算`
